share/completions/adduser.fish

```diff
@@ -11,18 +11,18 @@ complete -c adduser -l disabled-login -d 'Do not run passwd to set the password'
 complete -c adduser -l disabled-password -d 'Like --disabled-login, but logins are still possible (for example using SSH RSA keys) but not using password authentication'
 complete -c adduser -l force-badname -d 'By default, user and group names are checked against the configurable regular expression NAME_REGEX (or NAME_REGEX if --system is specified) specified in the configuration file'
```

Comment:
These two are also very long - this one's a whopping 176 characters!

Maybe "Disable login with password authentication" and "Allow usernames that don't pass restrictions"?

Reply:
Definitely too long! I've brought them down to 65 and 59 characters, respectively.